style.css

```diff
@@ -78,16 +40,60 @@
   color: #6070ff;
 }
 
-.hidden {
-  display: none;
-}
+/* NAV-menu */
+
+
 
 .show {
   width: 100%;
-  height: 100%;
+  height: 100vh;
   position: absolute;
-  background-color: #2230d2;
-  opacity: 0.8;
+  background-color: #6070FF;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(8px);
+}
```

Comment:
- [ ] When you show your menu on the mobile version, please make sure that the text is shown properly and you can distinguish the words. If you check the image below, you will see how your `I'm Mariana` text is being shown on your menu and not allowing the user to properly see the options, so please take a look at this issue.

Reply:
Thank you for the review. I have done the changes.